scripts/k6/buffer-static.js

```diff
@@ -0,0 +1,72 @@
+import { sleep } from "k6";
+import * as common from "./common.js";
+import { post, del } from "./common.js";
+
+export const options = common.options;
+
+const RECEIVER_ID = "buffer-static";
+
+export function setup() {
+  common.setup();
+
+  let res = del(`v1/receivers/${RECEIVER_ID}`);
+  if (res.status !== 200 && res.status !== 404) {
+    console.error(res);
+    throw new Error("failed to delete receiver");
+  }
+
+  res = post("v1/receivers", {
+    id: RECEIVER_ID,
+    name: "Buffer API Static Benchmark",
+    exports: [
+      {
+        exportId: "test-export",
+        name: "test-export",
+        mapping: {
+          tableId: "in.c-buffer-static.data",
+          columns: [
+            { type: "id", name: "id" },
+            { type: "datetime", name: "datetime" },
+            { type: "ip", name: "ip" },
+            { type: "body", name: "body" },
+            { type: "headers", name: "headers" },
+          ],
+        },
+      },
+    ],
+  });
+  if (res.status !== 200) {
+    console.error(res);
+    throw new Error("failed to create receiver");
+  }
+
+  const { url } = res.json();
+  const endpoint = url.slice(url.indexOf("v1"));
+
+  const data = { a: "b", c: { d: "e", f: { g: "h" } } };
+  const headers = {
+    "My-Custom-Header": "custom header value abcd",
+  };
+
+  res = post(endpoint, data, headers);
```

Comment:
@jprochazk viem nejak poslat viac tychto requestov?

Ak tomu spravne chapem, tak sa vzdy posle rovnaky pocet requestov na vytvorenie receiver a na import.

Vytvorenie receiver vytvara vsetky resources, to vieme, ze bude nejakych `10s`.

Skor nas zaujima ten import, ktory sa bude volat ovela viac,
tak ci sa to da nejak nasimulovat a iba som na to neprisiel, alebo to chyba.

Reply:
Nevím, jestli správně chápu. Tohle je jen setup funkce, tady se vytvoří receiver, a tady ten jeden import je tu jen kvůli debugu. Importy se posílají v rámci té `export default function` - to je hlavní "body" toho benchmarku. Takže se pošle jeden `POST /v1/receivers` request a pak N `POST /v1/import/...` requestů.